src/modules/Tab/Tab.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory
+} from "../../types";
+import { withStyles } from "../../base";
+
+export interface IProps extends StandardProps<"li"> {
+  disabled?: boolean;
+  index?: number;
+  label: string | JSX.Element;
+  selectedTab?: number;
+  onTabClick?: (index?: number) => void;
+}
+
+const styleSheet: ThemeStyleSheetFactory = theme => ({
+  tab: {
+    cursor: "pointer",
+    listStyle: "none",
+    marginRight: theme.spaceUnit.md,
+    maxWidth: "50px",
```

Comment:
This seems too restrictive, the tab label would have to be very short, it also isn't dealign with text overflow:

<img width="237" alt="Screenshot 2020-02-06 at 10 59 00" src="https://user-images.githubusercontent.com/5501872/73931298-f5f31480-48cf-11ea-8b49-309e582fa3c0.png">

Reply:
That's true, I removed that style. 
As for the text overflow, I'll leave that to the design systems to implement! (As little styling in diana as possible)